Algorithm.CSharp/CustomDataBitcoinAlgorithm.cs

```diff
@@ -41,8 +42,17 @@ public override void Initialize()
             //Set the cash for the strategy:
             SetCash(100000);
 
-            //Define the symbol and "type" of our generic data:
-            AddData<Bitcoin>("BTC");
+            // Define our custom data properties and exchange hours
+            var ticker = "BTC";
+            var properties = new SymbolProperties("Bitcoin", "USD", 1, (decimal)0.01, (decimal)0.01, "BTC");
+            var exchangeHours = SecurityExchangeHours.AlwaysOpen(TimeZones.NewYork);
+
+            // Add the custom properties and exchange hours to our database
+            SymbolPropertiesDatabase.SetEntry(Market.USA, ticker, SecurityType.Base, properties);
+            MarketHoursDatabase.SetEntry(Market.USA, ticker, SecurityType.Base, exchangeHours);
+
+            // Add the custom data to our algorithm
+            AddData<Bitcoin>(ticker);
```

Comment:
Cool! we should add these `Sets` in the python equivalent algorithm.
Note this algorithm isn't part of the regression test suite, maybe we could directly add a _new_ C#/Py algorithm regression algorithm asserting these `Sets` work as expected